src/data/faq.json

```diff
@@ -491,16 +491,30 @@
                     {
-                        "title": "[Google/Android] A card on my Start Screen is notifying me that my device is not or only partially compatible. What does this mean?",
-                        "anchor": "incompatibility_warning",
+                        "title": "[Google/Android] A card on my Start Screen is notifying me that my device is only partially compatible. What does this mean?",
+                        "anchor": "part_incompat",
                         "active": true,
                         "textblock": [
-                            "If you see such a notice, your device is not able to send any random IDs via Bluetooth. This can be caused by a restriction due to your combination of hardware and software on your device. Because of it, the Corona-Warn-App cannot use Bluetooth Low Energy in its so-called “peripheral mode”.",
-                            "In case you are infected and you decide to warn others, only users who were checked in to the same event or the same location, at the same time as you were, can be warned. Your own risk assessment is <b>not affected</b>. All other features of the app are also not affected.",
-                            "Please ensure that you are using the latest version of the operating system available for your smartphone. If the problem persists, please contact your device manufacturer.",
-                            "If the card shows that your device is completely incompatible, the cause could be that your device does not contain a Bluetooth chip. Only in this case, you cannot exchange random IDs with others and risk assessment can only base its calculation on locations and events you checked in to.",
-                            "<b>Both Check-in and Contact Journal functionality, as well as retrieving PCR and rapid test results, is fully functional without restriction in any case</b> and only proximity-based ID recording is affected.",
-                            "Bluetooth compatibility checking and warnings were introduced with Android Corona-Warn-App version 2.2."
+                            "Since version 2.2, the Corona-Warn-App for Android contains a check which determines whether your device is fully capable of all Bluetooth-related features which the app needs to operate normally. If you are seeing this message, the operating system of your device is lacking a functionality that Corona-Warn-App normally uses.",
+                            "Due to this, your device is not able to <i>send</i> any random IDs via Bluetooth. However, you are still able to <i>receive</i> random IDs. Therefore, <b>your own risk assesment is not affected</b>.",
+                            "The only limitation which this incompatibility causes is that, in case you are infected and you decide to warn others, only users who were checked in to the same event or the same location, at the same time as you were, can be warned. All other features of the app are not affected.",

```

Comment:
Suggest:
"All other features of the app are not affected." =>
"None of the other app features are affected."